sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -707,9 +707,9 @@ def begin_analyze(  # type: ignore
             list[dict[str, str]]
         :param tasks: A list of tasks to include in the analysis.  Each task object encapsulates the parameters
             used for the particular task type.
-        :type tasks: list[Union[~azure.ai.textanalytics.EntitiesRecognitionTask,
-            ~azure.ai.textanalytics.PiiEntitiesRecognitionTask, ~azure.ai.textanalytics.EntityLinkingTask,
-            ~azure.ai.textanalytics.KeyPhraseExtractionTask, ~azure.ai.textanalytics.SentimentAnalysisTask]]
+        :type tasks: list[Union[~azure.ai.textanalytics.EntitiesRecognitionAction,
+            ~azure.ai.textanalytics.PiiEntitiesRecognitionAction, ~azure.ai.textanalytics.EntityLinkingTask,
+            ~azure.ai.textanalytics.KeyPhraseExtractionAction, ~azure.ai.textanalytics.SentimentAnalysisTask]]
```

Comment:
I think maybe this must've gotten left over from when we previously had planned to release Sentiment for v3.1-preview.3, so we should probably remove it for now.